scripts/eurostat/life_expectancy_by_age_gender/preprocess.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+import re
+import sys
+path = 'demo_r_mlifexp.tsv'
+
+def group_to_wide_form(data, col):
+    """reorgnize the data of each statvars into independent columns """
+    data_grouped = data.groupby([col])
+    subsets = []
+    for _, subset in data_grouped:
+        pivot = subset[col].iloc[0]
+        subset = subset.rename(columns = {'geo': pivot+'_geo',
+            'year': pivot + '_year', 'life_expectancy': pivot})
+        subset = subset.drop(columns = [col]).reset_index(drop=True)
+        subsets.append(subset)
+    data = pd.concat(subsets, axis=1, join = 'outer')
+    return data
+
+def nuts_to_iso(data):
+    """convert nuts code to ISO Alpha3 code"""
+    # no need in the future, can be resolved through ISO Alpha 2
+    codes = pd.read_csv("./ISO_NUTS_country_map.csv")
+    code_dict = codes.set_index('NUTS').to_dict()['Alpha-3 code']
+    data['geo'] = data['geo'].map(code_dict)
+    assert (~data['geo'].isnull()).all()
+    return data
+
+def preprocess(filepath):
+    """preprocess the tsv file for import into DataCommons"""
+
+    data = pd.read_csv(filepath, sep='\t')
+
+    # concatenate columns of different years
+    identifier = 'unit,sex,age,geo\\time'
+    assert identifier in data.columns
+    years = list(data.columns.values)
+    years.remove(identifier)
+    data = pd.melt(data, id_vars=identifier, value_vars=years, 
+                   var_name='year', value_name='life_expectancy')
+
+    # convert string to numbers, drop the lables
+    data['year'] = data['year'].astype(int)
```

Comment:
I don't think you need to convert to int

Reply:
I want to get rid of the spaces appended to the end of the string e.g. "2018 "